spec/lib/bolt_spec/puppet_agent.rb

```diff
@@ -35,10 +35,18 @@ def install(target, collection: 'puppet6', inventory: nil)
         }
       }
       inventory ||= {}
-      result = run_task('puppet_agent::install', target,
-                        { 'collection' => collection },
-                        config: config,
-                        inventory: inventory)
+      # Task will get latest collection without collection specified
+      result = if collection
```

Comment:
The only conditional part of this is the params, right? So this can be simplified a bit

```
task_params = if collection
                { 'collection' => collection }
              else
                {}
              end

run_task('puppet_agent::install', target, task_params, config: config, inventory: inventory)
```